docs/api-docs/customers/customers-subscribers-overview.md

```diff
@@ -213,28 +222,27 @@ The Subscribers API allows you to manage subscribers who have signed up for the
+Password validation is only available on the V2 Customers API. Validation will return a boolean. The V3 Customers API can reset a customer's password or input a new password. 
 
 ## Related resources
 
 ### Articles
 - [Adding and Editing Fields in the Account Signup Form](https://support.bigcommerce.com/s/article/Editing-Form-Fields#account-fields)
-- [Checkout Settings](https://support.bigcommerce.com/s/article/Checkout-Settings#checkout-settings)
-- [Storefront Subscriptions](https://developer.bigcommerce.com/api-reference/storefront/storefront-subscriptions)
+- [Checkout Settings](https://support.bigcommerce.com/s/article/Optimized-Single-Page-Checkout#checkout-settings)
 
 ### Endpoints
-- [Customer Login API](https://developer.bigcommerce.com/api-docs/customers/customer-login-api)
-- [Current Customer API](https://developer.bigcommerce.com/api-docs/customers/current-customer-api)
-- [Customers API](https://developer.bigcommerce.com/api-reference/customer-subscribers/v3-customers-api)
-- [Customer Groups](https://developer.bigcommerce.com/api-reference/customer-subscribers/customers-api/customer-groups/getallcustomergroups) (Customer V2 API)
-- [Password Validation](https://developer.bigcommerce.com/api-reference/customer-subscribers/customers-api/customer-passwords/validatecustomerpassword) (Customer V2 API)
-- [Password Confirmation](https://developer.bigcommerce.com/api-reference/customer-subscribers/customers-api/customers/createanewcustomer) (Customer V2 API)
-- [Subscribers API](https://developer.bigcommerce.com/api-reference/customer-subscribers/subscribers-api)
+- [Customer Login API](https://developer.bigcommerce.com/api-docs/storefront/customer-login-api)
+- [Current Customer API](https://developer.bigcommerce.com/api-docs/storefront/current-customer-api)
+- [Customers V3 API](https://developer.bigcommerce.com/api-reference/store-management/customers-v3)
+- [Customer Groups](https://developer.bigcommerce.com/api-reference/store-management/customers-v2/customer-groups/getallcustomergroups) (Customers V2 API)
+- [Password Validation](https://developer.bigcommerce.com/api-reference/store-management/customers-v2/customer-passwords/validatecustomerpassword) (Customers V2 API)
+- [Password Confirmation](https://developer.bigcommerce.com/api-reference/store-management/customers-v2/customers/createanewcustomer) (Customers V2 API)
+- [Storefront Subscriptions](https://developer.bigcommerce.com/api-reference/storefront/storefront-subscriptions)
+- [Subscribers API](https://developer.bigcommerce.com/api-reference/store-management/subscribers)
 
 ### Webhooks
-- [Customers](https://developer.bigcommerce.com/api-docs/getting-started/webhooks/webhook-events#webhook-events_customer)
-
+- [Customers](https://developer.bigcommerce.com/api-docs/store-management/webhooks/events#customer)
```

Comment:
The link takes you to a sub-section called "Customer"
Maybe you should change it to 
- [Customer](https://developer.bigcommerce.com/api-docs/store-management/webhooks/events#customer)